src/main/webapp/scripts/session-script.js

```diff
@@ -1,16 +1,133 @@
-// Copyright 2019 Google LLC
```

Comment:
The changes here don't account for the new ServerClient, I don't know if it might be wise to just hold off?